card/lib/card/format/nest.rb

```diff
@@ -59,13 +59,13 @@ def reuse_format?
       # recursion is caught, view recursion is not.
       # TODO: catch view recursion and remove this. (Should be straightforward within voo)
       def nest_recursion_risk?
-        content_view? && format.voo&.structure
+        content_view? # && format.voo&.structure
```

Comment:
why would have to have a voo.structure to be a recursion risk??

Reply:
because the structure can nest the card itself?

Reply:
If having a voo.structure indicates it's a likely risk, maybe it should be `||`.

I was seeing recursions that weren't getting caught because there was no voo.structure.